docs/content/concepts/assets/asset-jobs.mdx

```diff
@@ -0,0 +1,106 @@
+---
+title: Asset jobs | Dagster
+description: Asset jobs are the main unit for materializing and monitoring Software-defined assets in Dagster.
+---
+
+# Asset jobs
+
+<Note>
+  Looking to execute a <a href="/concepts/ops-jobs-graphs/graphs">graph</a> of{" "}
+  <a href="/concepts/ops-jobs-graphs/ops">ops</a>, which aren't tied to
+  Software-defined Assets? Visit the{" "}
+  <a href="/concepts/ops-jobs-graphs/op-jobs">Op jobs</a> page.
+</Note>
+
+Jobs are the main unit for executing and monitoring Software-defined assets in Dagster. An asset job materializes a selection of [Software-defined Assets](/concepts/assets/software-defined-assets). It can be launched in a few different ways:
```

Comment:
now that we have a kinda overarching jobs page, should we also mention that page here for further info?

Reply:
Yeah, we can do that